luaui/Widgets/gui_info.lua

```diff
@@ -1050,7 +1050,8 @@ local function drawUnitInfo()
 		end
 
 		if unitDefInfo[displayUnitDefID].weapons then
-			local reloadTime = 1
+			-- reloadTime indicates percentage of reload time speedup
```

Comment:
Thanks for the comment, but seems `reloadTime` is a misleading name. The discrepancy made it hard to understand the comment and I ended up having to sift through the code.

Rename to something like `reloadSpeedup`?